lib/defaults/defaults.go

```diff
@@ -219,6 +219,9 @@ const (
 	// AgentStopTimeout is amount of time agent gets to gracefully shut down
 	AgentStopTimeout = 10 * time.Second
 
+	// AgentStatusTimeout specifies the timeout for agent status query
+	AgentStatusTimeout = 5 * time.Second
```

Comment:
I'm sort of on the fence with timeouts - I feel these should be configurable via envars at least to avoid any issues with slower networks and arbitrary cluster sizes.

Reply:
Gravity will now check for `GRAVITY_AGENT_STATUS_TIMEOUT` env var. Will default to 5sec if env var is not specified.